client/blocks/url-preview/index.jsx

```diff
@@ -76,7 +77,8 @@ export default function urlPreview( WebPreview ) {
 			}
 
 			return (
-				<WebPreview
+				<AsyncLoad
+					require="components/web-preview"
```

Comment:
I don't like how we need to remember to use `<AsyncLoad>` at every place we want to use the `component/web-preview` component. One day, someone will forget about that and will import `component/web-preview` directly.

What if we encapsulated the async loading into the `component/web-preview` module? The `index.jsx` would contain the async load:
```jsx
// index.jsx
export default props => (
  <AsyncLoad
    require="components/web-preview/component"
    { ...props }
  />
);
```
and the current `index.jsx` would be renamed to `component.jsx`.

Then all the usage sites would just import the `WebPreview` component as they always did:
```jsx
import WebPreview from 'components/web-preview';

<WebPreview ... />
```

The async loading would become a hidden implementation detail, transparent to the user of the component.

Reply:
That's a really good suggestion, thanks!

Reply:
Awesome idea @jsnajdr. I love it